samples/matter/common/src/thread_util.cpp

```diff
@@ -43,3 +56,21 @@ void StartDefaultThreadNetwork(uint64_t datasetTimestamp)
 	chip::app::DnssdServer::Instance().StartServer();
 #endif
 }
+
+#ifdef CONFIG_MPSL_FEM
+CHIP_ERROR SetDefaultThreadOutputPower()
+{
+	CHIP_ERROR err;
+	/* set output power when FEM is active */
+	if (CONFIG_FEM_802_15_4_DEFAULT_TX_POWER != 0 &&
+	    CONFIG_FEM_802_15_4_DEFAULT_TX_POWER >= kMinThreadOutputPower &&
+	    CONFIG_FEM_802_15_4_DEFAULT_TX_POWER <= kMaxThreadOutputPower) {
+		err = chip::DeviceLayer::Internal::MapOpenThreadError(
```

Comment:
You should use `LockThreadStack`/`UnlockThreadStack` before and after this call.